src/graph/serializers/graphcontext_type.c

```diff
@@ -9,6 +9,7 @@
 #include "graphcontext_type.h"
 #include "serialize_graph.h"
 #include "serialize_store.h"
+#include "../../util/arr.h"
```

Comment:
I've disabled/deleted the persistency test,
Let's bring it back and make sure it passes